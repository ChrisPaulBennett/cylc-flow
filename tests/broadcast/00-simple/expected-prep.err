ERROR: No broadcast to cancel/clear for these options:
  --cancel=[environment]BCAST
  --namespaces=m4
  --point_strings=2010080900
ERROR: No broadcast to cancel/clear for these options:
  --cancel=[environment]BCAST
  --namespaces=m5
  --point_strings=all-cycle-points
ERROR: No broadcast to cancel/clear for these options:
  --namespaces=m6
ERROR: No broadcast to cancel/clear for these options:
  --namespaces=m7
ERROR: No broadcast to cancel/clear for these options:
  --namespaces=ENS3
