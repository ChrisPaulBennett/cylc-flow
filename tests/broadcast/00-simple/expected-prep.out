Broadcast set:
+ [root.all-cycle-points] [environment]BCAST=ROOT
Broadcast set:
+ [foo.2010080800] [environment]BCAST=FOO
Broadcast set:
+ [bar.all-cycle-points] [environment]BCAST=BAR
Broadcast set:
+ [baz.2010080900] [environment]BCAST=BAZ
Broadcast set:
+ [qux.2010080900] [environment]BCAST=QUX
Broadcast cancelled:
- [qux.2010080900] [environment]BCAST=QUX
Broadcast set:
+ [wibble.all-cycle-points] [environment]BCAST=WIBBLE
Broadcast cancelled:
- [wibble.all-cycle-points] [environment]BCAST=WIBBLE
Broadcast set:
+ [ENS.all-cycle-points] [environment]BCAST=ENS
Broadcast set:
+ [ENS1.all-cycle-points] [environment]BCAST=ENS1
Broadcast set:
+ [m2.2010080900] [environment]BCAST=M2
Broadcast set:
+ [m7.all-cycle-points] [environment]BCAST=M7
Broadcast set:
+ [m8.all-cycle-points] [environment]BCAST=M8
Broadcast set:
+ [m9.all-cycle-points] [environment]BCAST=M9
2010080800         
 `-foo             
   `-environment   
     `-BCAST       FOO
2010080900         
 |-baz             
 | `-environment   
 |   `-BCAST       BAZ
 `-m2              
   `-environment   
     `-BCAST       M2
all-cycle-points   
 |-ENS             
 | `-environment   
 |   `-BCAST       ENS
 |-ENS1            
 | `-environment   
 |   `-BCAST       ENS1
 |-bar             
 | `-environment   
 |   `-BCAST       BAR
 |-m7              
 | `-environment   
 |   `-BCAST       M7
 |-m8              
 | `-environment   
 |   `-BCAST       M8
 |-m9              
 | `-environment   
 |   `-BCAST       M9
 `-root            
   `-environment   
     `-BCAST       ROOT
